composer.json

```diff
@@ -36,15 +37,17 @@
         "league/oauth2-instagram": "@stable",
         "league/oauth2-google": "@stable",
         "league/oauth2-linkedin": "@stable",
-        "google/recaptcha": "@stable"
+        "google/recaptcha": "@stable",
+        "robthree/twofactorauth": "^1.5.2"
```

Comment:
@steinkel , I've removed it from `require` param, and left it only in `require-dev` just to make sure that unit tests will pass.